Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int user = 0; user < 2; user++)
+            {
+                string name = ConsolePromptToString("Name: ");
```

Comment:
Consider moving this to its own method

Reply:
which part do you mean? can you be more specific?